runner.sh

```diff
@@ -5,4 +5,13 @@ python evaluation/main_nlg_prompt_batch.py tha $MODEL_NAME 0 4
 python evaluation/main_llm_judge_batch.py $MODEL_NAME --data ThaiLLM-Leaderboard/mt-bench-thai
 python scripts/transform_result.py $MODEL_NAME
 # clear model cache
-rm -rf ~/.cache/huggingface/hub/
\ No newline at end of file
+rm -rf ~/.cache/huggingface/hub/
+
+# run with OpenAI API compatible
```

Comment:
Please move this section into new file and specific that on README.md  eval_openai_compatible.sh

Reply:
also please add information that due to openai compatible cannot access logits, the result on nlu task is not directly comparable to the official leaderboard